.gitignore

```
@@ -3,10 +3,15 @@ build
 debug
 __pycache__
 catalyst.egg-info
```

Just noticed this is out of date so we can fix it here:
```suggestion
pennylane_catalyst.egg-info
```